src/System.Runtime.Extensions/tests/System/Environment.Exit.cs

```diff
@@ -42,6 +42,15 @@ public static void CheckExitCode(int expectedExitCode)
 
         [Theory]
         [MemberData(nameof(ExitCodeValues))]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapNotUapAot, "RemoteExecutor behaves different in UAP.")]
```

Comment:
@safern, I still don't understand... why would you need to access the Process object directly at all?  With the new ExpectedExitCode you added, why isn't the whole test just something like:
```C#
var options = new RemoteInvokeOptions { ExpectedExitCode = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? expectedExitCode : (int)(sbyte)expectedExitCode };
RemoteInvoke(s => int.Parse(s), expectedExitCode.ToString(), options).Dispose();
```
?